pom.xml

```diff
@@ -6,12 +6,12 @@
 
   <groupId>us.warframestat</groupId>
   <artifactId>moddetection-api</artifactId>
-  <version>1.0.0</version>
+  <version>1.0.0.1</version>
```

Comment:
Non blocker: Want to keep to Semantic Versioning?

Reply:
ideally, yeah, so next up should be 1.1.0... this should get incremented by the ci release, right?